Xamarin.ExposureNotification/ExposureNotification.ios.cs

```diff
@@ -72,30 +77,58 @@ static Task PlatformScheduleFetch()
 
 			void scheduleBgTask()
 			{
+				if (ENManager.AuthorizationStatus != ENAuthorizationStatus.Authorized)
```

Comment:
The user has not permitted this, so it will popup. Also, if the user does not want this, then no need to schedule.

Reply:
This means the app should later try and schedule again... After the user is prompted and does enable it?